src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Utilities/NullableHelpers/NullableExtensions.cs

```diff
@@ -11,9 +11,17 @@ namespace Microsoft.CodeAnalysis
     internal static partial class NullableExtensions
     {
         public static ITypeSymbol? GetConvertedTypeWithAnnotatedNullability(this TypeInfo typeInfo)
-            => typeInfo.ConvertedType?.WithNullableAnnotation(typeInfo.ConvertedNullability.Annotation);
+            => typeInfo.ConvertedType
+#if !CODE_STYLE // TODO: Remove the #if once WithNullableAnnotation is available.
```

Comment:
`.WithNullableAnnotation` and `.NullableAnnotation` are not yet available in CodeStyle layer. Follow-up work item tracks reverting these changes once they are available.